src/RNGalleryList.ts

```diff
@@ -36,6 +37,16 @@ import {WindowsHelloExamplePage} from './examples/WindowsHelloExamplePage';
 import {ExpanderExamplePage} from './examples/ExpanderExamplePage';
 import {VirtualizedListExamplePage} from './examples/VirtualizedListExamplePage';
 
+let RNGalleryCategories = [
```

Comment:
Did you rename "Text" to "System"?

Reply:
Not intentionally. "System" was supposed to be an addition. But something got messed in the merge from my other branch and "Text" got replaced. Will fix.